services/blockchain-indexer/shared/dataService/business/pos/stakers.js

```diff
@@ -25,21 +25,21 @@ const stakesIndexSchema = require('../../../database/schema/stakes');
 const {
 	updateAccountPublicKey,
 	getIndexedAccountInfo,
-	getAccountsTable,
+	getAccountsTableReplica,
 	getLisk32AddressFromPublicKey,
-} = require('../../../utils/account');
+} = require('../../../indexer/utils/account');
 
-const MYSQL_ENDPOINT = config.endpoints.mysql;
+const MYSQL_ENDPOINT_REPLICA = config.endpoints.mysqlReplica;
 
 const getStakesTable = () => getTableInstance(
 	stakesIndexSchema.tableName,
 	stakesIndexSchema,
-	MYSQL_ENDPOINT,
+	MYSQL_ENDPOINT_REPLICA,
 );
 
 const getStakers = async params => {
 	const stakesTable = await getStakesTable();
-	const accountsTable = await getAccountsTable();
+	const accountsTableReplica = await getAccountsTableReplica();
```

Comment:
to indicate that this uses a replicate connection. This would help to code so we would not need to see the connection definition every time reading the code. will revert to previous naming